hazelcast/src/test/java/com/hazelcast/internal/metrics/metricsets/ClassLoadingMetricSetTest.java

```diff
@@ -31,16 +31,21 @@
 import java.lang.management.ClassLoadingMXBean;
 import java.lang.management.ManagementFactory;
 
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.CLASSLOADING_JET_EXTENSIONS_PREFIX;
 import static com.hazelcast.internal.metrics.MetricDescriptorConstants.CLASSLOADING_FULL_METRIC_LOADED_CLASSES_COUNT;
 import static com.hazelcast.internal.metrics.MetricDescriptorConstants.CLASSLOADING_FULL_METRIC_TOTAL_LOADED_CLASSES_COUNT;
 import static com.hazelcast.internal.metrics.MetricDescriptorConstants.CLASSLOADING_FULL_METRIC_UNLOADED_CLASSES_COUNT;
 import static com.hazelcast.internal.metrics.ProbeLevel.INFO;
+import static com.hazelcast.internal.metrics.metricsets.ClassLoadingMetricSet.JET_EXTENSION_TO_CLASS_MAPPING;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(QuickTest.class)
 public class ClassLoadingMetricSetTest extends HazelcastTestSupport {
 
+    private static final String JET_EXTENSION_AVAILABLE_IN_CLASSPATH = "avro";
```

Comment:
To avoid the dependency cycle you can do either 
-  move this test to `hazelcast-it` module (you would need to create a submodule there, as the current ones are not really suitable for this kind of test), this module would depend on `hazelcast` and `hazelcast-jet-avro`, without creating a cycle
- alternatively, you could use one of the 3rd party classes, e.g. parquet, and add a dependency on parquet, but this will pollute the test classpath of all core tests. I very much prefer a separate test module with its own classpath.